### PR TITLE
updated im dep + image

### PR DESCRIPTION
**Pull request by joshimoo**
wait for newly released im image before merge
depends on https://github.com/longhorn/longhorn-instance-manager/pull/115
longhorn/longhorn#3966

### CODE REVIEW

**Comment by joshimoo** [comment]
Wait for im PR merge + image release then update the commits to the released image/commits.
https://github.com/longhorn/longhorn-instance-manager/pull/115

**Comment by innobead** [comment]
LGTM, and waiting for the update after the IM PR merge. Also, remember to update longhorn/longhorn as well.

**Review by innobead — APPROVED** [review]

**Pull request merged**
